sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryKMP.kt

```diff
@@ -4,10 +4,33 @@ import io.sentry.kotlin.multiplatform.protocol.SentryId
 import io.sentry.kotlin.multiplatform.protocol.UserFeedback
 
 typealias ScopeCallback = (Scope) -> Unit
+typealias OptionsConfiguration = (SentryOptions) -> Unit
+
+expect abstract class Context
```

Comment:
Why do we need that?

Reply:
this is a rather 'elegant' I'd say workaround to avoid `Any` context.

`expect abstract class` expects us to implement this in every submodule.

so on androidMain we say `actual typealias Context = Context (android context)`

on Cocoa it's just an empty class: `actual abstract class Context`

This works because the expected class Context doesn't have a body.

Now if you try to initialize the Android SDK, you can only pass in the Android Context and nothing else which is much better than using `Any`